.github/workflows/ci_check_dependencies.yml

```diff
@@ -47,14 +47,16 @@ jobs:
       python_version: "3.9"
       install_extras: "dev"
       pr_labels: "CI/CD"
+      # Follow issue EMMC-ASBL/oteapi-services#328 for more information about the
+      # urllib3 dependency.
       ignore: |
         dependency-name=agraph-python...version=>=103
         dependency-name=celery...version=>=6
         dependency-name=diskcache...version=>=6
         dependency-name=numpy...version=>=2
         dependency-name=openpyxl...version=>=4
         dependency-name=Pillow...version=>=11
-        dependency-name=pydantic...version=>=2
```

Comment:
version 3?

Reply:
Yeah. This job is for the "standard" pydantic v2 development branch. So we up the ignore rule to ignore pydantic v3. We _want_ updates to new v2 versions :)